slackbridge/bots.py

```diff
@@ -89,6 +89,18 @@ def check_slack_rtm(self):
             elif message['presence'] == 'active':
                 user_bot.back()
             return
+        elif (message['type'] == 'member_joined_channel' and
+                message['user'] in self.users):
+            self._rtm_channel_helper(message)
+            return
+        elif (message['type'] == 'member_left_channel' and
```

Comment:
I'd surround this condition and the two above it with `message['user'] in self.users` since they all check that independently when there could just be one condition for it. Maybe put the `team_join` check at the start so that you can group all the methods that need a valid user after it?